src/app/button/button.component.scss

```diff
@@ -0,0 +1,6 @@
+button {
+    border: solid 1px;
```

Comment:
color?

Reply:
I went with default black border and made the whole button lightskyblue.